frontend/lib/search/result_list.dart

```diff
@@ -0,0 +1,83 @@
+import 'dart:math';
+
+import 'package:ehrenamtskarte/graphql/graphql_api.graphql.dart';
+import 'package:ehrenamtskarte/map/detail/detail_view.dart';
+import 'package:ehrenamtskarte/widgets/error_message.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:graphql_flutter/graphql_flutter.dart';
```

Comment:
```suggestion
import 'package:flutter/cupertino.dart';
import 'package:flutter/material.dart';
import 'package:graphql_flutter/graphql_flutter.dart';

import '../graphql/graphql_api.graphql.dart';
import '../map/detail/detail_view.dart';
import '../widgets/error_message.dart';
```

When possible, you should use the relative path according to the Dart style guide.

Reply:
I removed cupertino.
The relative path thing is a bit tricky, as Intellij doesn't do that out of the box. Hence it's not practical to keep up such a rule i think

Reply:
Yep, you are right. I also misunderstood the style guide, one only should use the relative path if the other file is _in the same directory_.